queryset_sequence/__init__.py

```diff
@@ -493,7 +493,11 @@ def __init__(self, model=None):
             self.DoesNotExist = ObjectDoesNotExist
 
     def __getattr__(self, name):
-        return getattr(self._model, name)
+        if name == "_meta":
+            return getattr(self._model, name)
+        if hasattr(super(), "__getattr__"):
+            return super().__getattr__(name)
+        raise AttributeError(name)
```

Comment:
So with this change the *only* thing being proxied is `_meta`. Would it make more sense to define only that?

```python
@property
def _meta(self):
    return getattr(self._model, "_meta")
```